src/editor/Editor.js

```diff
@@ -959,7 +961,9 @@ class Editor extends EditorStartup {
   * @returns {void} Resolves to `undefined`
   */
   cancelOverlays (e) {
-    $('#dialog_box').hide();
+    if($id("dialog_box") != null){
```

Comment:
maybe you can use the new ? operator to make this more readable. 
$id("dialog_box")?.style.display = 'none'

Reply:
Modified now please check it.